docs/operations.md

```diff
@@ -49,3 +49,31 @@ The `ReplicaSet` for each Etcd node ensures that the cluster can function even w
          by the location of the existing `PersistentVolumeClaim`and its bound `PersistentVolume`.
    3. When the server has rebooted, Kubernetes will be able to schedule the `Pod` and it will start.
       The Etcd process inside the `Pod` will use the existing data and it will rejoin the Etcd cluster.
+
+### Delete a Cluster
```

Comment:
worth including some docs on how to fully delete the cluster, including its data?